### DISCUSSION TITLE
How to set up OneDriveGUI with custom build of onedrive

### DISCUSSION THREAD

**Discussion by fobban** [bpozdena/OneDriveGUI#170] [General]
Hi,

first of all - great work with OneDriveGUI project. Got the question if there is some way to serup this with custom build of onedrive binary? I have a custom build provided by my company and need some way to integrate it with GUI. GUI loads, can open the OneDrive directory, binary is available in standard path under /usr/bin/onedrive, but GUI reports it is not installed. 
`onedrive --version
onedrive 2.4.25-5`
I am running Ubuntu 22.04.4 LTS with KDE 5.24.7.
![image](https://github.com/bpozdena/OneDriveGUI/assets/6831798/1b7abd6d-11db-463a-ae93-95bdcf78a139)
Is there any special check in the code that perhaps can be disabled for my use?

BR//Lukasz

**Comment by bpozdena** [comment]
Hi @fobban , 
can you please provide a screenshot of your terminal after you executing the below two commands?:
```
which onedrive
onedrive --version
```

Also please specify the version of your OneDriveGUI Appimage.

**Reply by fobban** [reply]
Hey,
sure, below is the output of two commands you asked:

which onedrive    
/usr/bin/onedrive

onedrive --version
onedrive 2.4.25-5eri0

And  the OneDriveGUI is  OneDriveGUI-1.1.0rc1-x86_64.appimage

**Comment by bpozdena** [comment]
Please read past issues, read description of files you are downloading, read documentation, read in general before asking anything anywhere.
